### PR TITLE
Video listing page

### PR DESCRIPTION
**Pull request by RahulSingh9131**
This branch contains
1. components for video page
2. styles
3. videos list in db

image for reference
videoPage

### CODE REVIEW

File: src/components/VideoCard.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { WatchLaterOutlined } from '@mui/icons-material';
+import ThumbUpOutlinedIcon from '@mui/icons-material/ThumbUpOutlined';
+import { Avatar } from '@mui/material';
+import React from 'react';
+import "../css/main.css";
+
+function VideoCard({_id,image,title,views,timestamp,category}) {
+  return (
+    <div className='videocard' key={_id}>
+        <img className='videocard-thumbnail' src={image} alt=""/>
```
Comment (nishasen) [inline src/components/VideoCard.js]: give some text in alt

File: src/components/VideoCard.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        <div className='videocard-info'>
+            <Avatar className="videocard-avatar" src={image} alt={image}/>
```
Comment (nishasen) [inline src/components/VideoCard.js]: Its better if you give some text in alt or some separate picture

File: src/css/screens/videocard.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    margin-bottom: 5px;
+    font-size: var(--normal-font-size);
+}
+.videocard-text p{
+    color: var(--text-color-light);
+    font-size: var(--small-font-size);
+}
+.videocard-avatar{
+    height: 30px !important;
+    width: 30px !important;
```
Comment (nishasen) [inline src/css/screens/videocard.css:31]: I like how you are giving imp to height and width

File: src/App.js [hunk 4]
```diff
@@ -2,12 +2,14 @@ import "./App.css";
...
 
 
 function App() {
   return (
     <div className="App">
       <Routes>
+        <Route path="/videopage" element={<VideoPage/>}/>
         <Route path="/mock-api" element={<MockAPI/>}/>
         <Route path="/" element={<Homepage/>}/>
       </Routes>
```
Comment (therajatg) [inline src/App.js:15]: you can have all routes in a separate file then u can import that file in App.js

**Review by therajatg — APPROVED** [review]
everything else looks good

File: src/pages/VideoPage.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+function VideoPage() {
+
+  const [video,setVideo]=useState([]);
+
+  const fetchVideo= async ()=>{
+    try{
+      const res= await axios.get("/api/videos")
+      setVideo(res.data.videos)
+    }catch(e){
+      alert("please check the api endpoints again...")
```
Comment (nishasen) [inline src/pages/VideoPage.js]: Give some user friendly alert here

**Review by nishasen — APPROVED** [review]
I liked how you have organized everything, small changes suggested. Good and neat work.

**Review by srejitk — APPROVED** [review]
No changes suggested. Clean code and Intuitive Classnames

**Pull request merged**